lib/tasks/remove_advisory.rake

```diff
@@ -0,0 +1,68 @@
+namespace :remove_advisory_from_editions do
+  desc "Process advisory govspeak in published editions"
+  task published_editions: :environment do
+    regex = Regexp.new(/(^@)([\s\S]*?)(@?)(?=(?:^\$CTA|\r?\n\r?\n|^@|$))/m).to_s
```

Comment:
There's an `EmbeddedContentPatterns` module in Whitehall that contains regex's used by the various extractor classes in lib/Govspeak. I think we should follow the same pattern here rather than defining the regexp in two files.

i.e. create a `Govspeak::AdvisoryPattern` module and include it in both your new service and your rake task